vernac/comAssumption.ml

```diff
@@ -225,8 +233,10 @@ let context_nosection sigma ~poly ctx =
       Monomorphic_entry Univ.ContextSet.empty
   in
   let fn subst d =
-    let (name,b,t,_impl) = context_subst subst d in
+    let (name,b,t,impl) = context_subst subst d in
```

Comment:
Why this change? I don't see a use of `impl`.

Reply:
It is an unwanted change which I forgot to revert. Thanks for noticing.

Nevertheless, it raises the question of what to do when `Context` occurs not in a section. Currently, it silently turns the declarations into `Axiom`/`Definition`. Should it, like `Variable`, raise a warning "Interpreting this declaration as if a global declaration prefixed by Local"? This is at least what the reference manual suggests in saying that `Context` "declare[s] variables in the context of the current section, like `Variable`". 

In any case, the `impl` should not be silently dropped when not in a section. It should be tested and a warning be raised if it is `Explicit` (i.e. if the user wrote `Context {x : T}` outside a section).